shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/TripDetailsStopList.kt

```diff
@@ -316,8 +320,9 @@ data class TripDetailsStopList(val stops: List<Entry>) {
                 while (scheduleIndex in stopIds.indices && predictionIndex in predictions.indices) {
                     val stopId = stopIds[scheduleIndex]
                     val prediction = predictions[predictionIndex]
-                    check(Stop.equalOrFamily(stopId, prediction.stopId, globalData.stops)) {
-                        "predictions=$predictions stopIds=$stopIds predictionIndex=$predictionIndex scheduleIndex=$scheduleIndex"
+                    if (Stop.equalOrFamily(stopId, prediction.stopId, globalData.stops)) {
+                        scheduleIndex--
+                        continue
                     }
```

Comment:
Since `check(true)` succeeds and `check(false)` throws, shouldn't this be `if (!Stop.equalOrFamily(...`?

Reply:
Oh, yes definitely. Weird that it seemed to be working okay with that 🤔